cfgov/v1/util/util.py

```diff
@@ -79,9 +79,9 @@ def get_secondary_nav_items(request, current_page):
     nav_items = []
     for sibling in pages:
         if page.id == sibling.id:
-            sibling = page.get_appropriate_page_version(request)
```

Comment:
Oops, I can remove the `else` since `sibling` doesn't need to get assigned to `sibling` obviously.   It does seem as if the if/else block isn't doing anything, but I didn't actually change that aspect of it.  If `page` and `sibling` are the same thing in the first case, then `page.get_appropriate_page_version` and `sibling.get_appropriate_page_version` should've returned the same thing, right?  Not opposed to removing it but just curious why it's like this at all.